### ISSUE TITLE
EM feedback

### ISSUE THREAD

**Issue by davelab6** [davelab6/matd-dissertation#1]
1. more reasons **why** fonts should be free! 
   - too much focus of digitality ("because type designs are just data, and data should be free in a free society") not about functionality
   - "should anything functional be given away for free?" 
   - the users need freedom to control the functionality: "either they control the thing, or the thing controls them"
   - if a work is published, then unlimited redistribution by individuals inevitable, and its unethical to punish people for sharing like that
   - for works of entertainment, it can make sense to restrict commercial use and distribution, but for functional works, it doesnt make sense, because businesses are about performing functions. 
   - "isn't music a business? isn't it performing a function? Of making people happy?" 
   - you cant say 'this music is broken', you can say 'i dont like it'
   - "a music recording may be broken - what if it was mixed horribly, or is only in mono, and you want it in stereo?"
   - many things are not subject to copyright because they are functional, this is a well established categorisation in law - so more about that from other sources, eg judges/courts
2. say more about this "free society" thing...
   - "what I want to know more about is just this concept of a 'Free society' that you kept mentioning... I want an intro that frames the current time/place/society and the society where these libre works would fit in. I was assuming that you were talking about a "free society" where people are sharing their time like this as some sort of future socialist utopia or something."
   - Referring to the success of Wikipedia is a good, relatable way to explain 
   - free society = http://www.un.org/en/documents/udhr/ 
   - latest dasht stuff about 1930s politics
   - disambiguate from libertarian stuff in https://en.wikipedia.org/wiki/Free_society
   - more on the historical context of what a free society is; and a sociological explanation about where rms is coming from, which i think is important. having spend the last 4 years mostly in the usa, i think i have a much better understanding of stuff like _American Nations_ ![American Nations](http://www.washingtonpost.com/blogs/govbeat/files/2013/11/upinarms-map.jpg)
   - the map in https://en.wikipedia.org/wiki/Stateless_society might also be relevant ![stateless societies](https://upload.wikimedia.org/wikipedia/commons/thumb/6/67/World_1000_BCE.png/640px-World_1000_BCE.png)
   - "did the early hackers that you referred to, or the GNU people etc. have any end-goal in mind? What were their motivations for doing this? What was the reality of their situation. maybe peoples' motivations are unknown really, but I'd be curious to see if they all had similar ideas on politics or whatever"
   - NYT idea of stallmanism. a bit in the oreilly biography of richard that i found particularly funny, near the start around the interview with his mom, and she's like 'richard was a stubborn kid'; and he's like "i learned to read early, and mom took me to the library, and i'd read about american history, about jefferson and peoples rights and all that. and i expected those ideas to apply to me." But he's like _eight years old_. We could say its "stallmanism," but acutally, back to the first map, I think of it as a kind of americanism. Or at least yankeedomism as per http://www.tufts.edu/alumni/magazine/fall2013/features/up-in-arms.html . The early computer culture of the 50s-80s was only in certain areas of the usa, mainly yankeedom and left coast.
   - the radical politics of ordinary people. In media cultlure, radio/tv, there's a "neutral" american accent
     which is a midlands accent, and there's a phrase in the midlands summary which for me is a key idea of american culture: "society should be organized to benefit ordinary people". and i think that is really are the core of the filesharing and libre culture: ordinary people use the state power to grant copyright monopolies over some kinds of works to benefit the ordinary people, not stuff like http://www.reuters.com/article/2011/12/20/idUS93817875020111220 . But this kind of thing isn't very important; personal file sharing is not a big deal like it was 10 years ago. 
   - jailing a filesharer isn't bad because it was a big, wealthy corportation that produced the film; if its was some busker who sells CDs in the subway it would also be bad for a busker to try to to put that guy in jail for sharing copies. sharing is good, if you believe in humanity's fundemental equality. of course, if you believe some people are better than others, some people are entitled to nice things and others dont deserve them, then unlimited sharing would not be good. 
   - "why business models that run on making something once and then selling copies, is seen as a bad/evil thing. why can't someone buy an extra copy and give it to someone"
   - making digital copies doesnt naturally cost anything, so on what principle do you propose an artificial cost for making copies? the public wants nice things; fine. but there are many ways to get to the nice things that dont require gilberto sanchez to go to jail. 
   - "but what is the incentive for the film company to make the crazy insane special-effects-laden xmen movie if people should be able to distribute it for free? will all movies now be made on $200 budgets? donation-only theatres?"
   - no, the business model of "selling seats in a room that is playing the movie on a big screen" doesnt change because of unlimited filesharing. people often go to the cinema for movies they could watch at home. and most movies are profitable from box office ticket sales alone; the xmen wolverine movie that gilberto sanchez went to jail for a year for had a budget of $150 million and box office take of$373 million, so it was already profitable. plus there is revenue in licensing commercial use, like airlines. tv networks will probably go away, but rightsholders can license commercial distribution, ie netflix streaming. so i dont see all movies now be made on $200 budgets, not at all.
   - and if it became unprofitable? well, what is the incentive today for musicians to make crazy insane baroch music? times change. i guess this is where the Far West culture comes in, the utopian vibe, being completely willing to imagine a society arranged differently with different outcomes. the point is that the public's well being has primacy; that democracy is about the great majority of people using their mass to balance the power of wealthy people, and that while society is organized under a system of capitalism (which is a temporary stage) the state's rules of business determine which business models are permitted. so the question is, are those rules benefitting the ordinary people, or do they harm the ordinary people? environmentalism is one aspect of that, because in an effective democracy, the public will order the state to restrict business models that physically harm the public by damanging the environment. filesharing is the opposite way, because of its a natural quality that digital copies cost nothing. 
   - make sure that its clear I don;t assume that all wealthy people are wealthy because they did something unethical to get there. its not that they accumulated capital in an unethical way (given surplus value capitalism is ethical) its that, having accumulated it, they can spend it to influence the state in ways that benefit them and harm the public. the central key idea of democracy is to prevent that. Although I think usually its between the A crowd and the B crowd, not the masses below - eg, magna carta in england or us revolution against the uk, it was a bunch of rich white guys saying oi to the richest white guy.
   - in an effective democracy, the public would order the state to "restrict" business models (that relied on artificial restraints on copying) by no longer imposing the artificial restraint. The restraint made sense before computer networks, and it can be hard to think about what things were like in the past. look at mobile phones from 10 years ago - https://www.google.com/search?q=mobile+phone+2005&tbm=isch -  this is a marshall mcluhan idea, that our media seep into our consciousness, and pervasive computer networks go very deep i think. So, in the past the public gave away its natural right to copy information, and grant publishers a monopoly on copying information through the state. But now the state is not responding to the public, but to the publishers; so the public trading away its natural right to copy information doesn't make sense any more, but publishers influence the state to make copyright monopolies stronger, not weaker. 
   - will this apply to physical things? not until people invent physical copiers. the history of typography is the history of copying information faster and faster, cheaper and cheaper. in a way physics is just information, a star trek beamer is a copier. 
   - how to make democracy effective is a big hard question, richard doesnt offer ideas about that. i think transparency is good - ie, more information published to the public - and computer networks are good at that. we see this already with the police brutality videos, and wikileaks. transparency has problems; the central example is the effect of transparency on trans people, who need opacity to literally survive. but i think the answer to "who watches the watchers?" is, the public. The satire of The Circle's 4k streaming politicians from a pin badge is not needed; chatnam house rules are used for a good reason. but corruption can be dealt with when state's operational information is more public. 
3. better practical, step by step, examples of how makers of libre works can earn money
   - making manuals/explaining the software/giving customer support is dissatisfying
   - update with the https://speakerdeck.com/davelab6/libre-planet-2015-free-fonts?slide=44 stuff
   - "how are people are contributing to it? Are they gaining any direct benefit (rather than indirect) from their contributions? i hate to make it about money, but yeah, I would love to just making libre fonts that underserved communities can really use as my job"
   - "Why was Red Hat able to make millionaires out of people?"

**Comment by davelab6** [comment]
In a business you can take revenues 'directly' or on the 'front end' by selling a product/service to the people who use it; a grocery store, for example, sells apples to people who eat them.

A business can also take revenues 'indirectly' on the 'back end' by giving away a product/service to the people who use it, and selling a related product/service to the same people or to other people; a magazine, for example, gives away the paper object (or lets their distributor charge a nominal fee) and sells advertising; or www.recurse.com gives away 12 weeks computer science tuition in NYC without a tuition fee, and sells referral fees to companies who hire their graduates. 

Front end revenue tends to be more speculative; you have to buy or grow the apples before you can sell them, and you may not be able to sell them and have them rot on you. 

Back end revenue tends to be less speculative; you take the advertisers fees before the magazine content is written, or you take the hiring companies money on a retainer basis. 

In capitalist society, there are 'open secrets' about how business works, such that most people work as employees where they consent to capitalists not paying them the full value of their work. The "economics of complementary goods" sounds very complicated, and dressing it up in language like that helps to obscure it from regular people like us, so that you tell me 'I have never seen an indirect business model in my life, they do not exist, I don't get it' :)

http://www.joelonsoftware.com/articles/StrategyLetterV.html covers this in some detail.

**Comment by davelab6** [comment]
> negative liberty could be defined as freedom from coercion and positive liberty as the opportunity to strive to fulfill one's potential.

https://en.wikipedia.org/wiki/The_Trap_(TV_series)#3._.22We_Will_Force_You_To_Be_Free.22_.2825_March_2007.29

**Comment by blankenfelde** [comment]
> and if it became unprofitable? well, what is the incentive today for musicians to make crazy insane baroch music? times change.

the musician who make crazy insane baroch music who is doing it between all their insecure minijobs cafe bar work temporary works to … not feel insane..!

**Comment by davelab6** [comment]
Right, people have intrinsic motivations outside of selling labour time
